kong/clustering/data_plane.lua

```diff
@@ -74,11 +74,20 @@ function _M.new(clustering)
 end
 
 
+local function set_control_plane_connected(reachable, ttl)
+  local ok, err = ngx.shared.kong:safe_set("control_plane_connected", reachable, ttl)
+  if not ok then
+    ngx_log(ngx_ERR, _log_prefix, "failed to set controlplane_reachable key in shm to ", reachable, " :", err)
```

Comment:
nit: update the log line to match the current name of the SHM key

```suggestion
    ngx_log(ngx_ERR, _log_prefix, "failed to set \"control_plane_connected\" key in shm to ", reachable, " :", err)
```